src/components/ui/phone-input.tsx

```diff
@@ -120,7 +120,11 @@ const CountrySelect = ({
           />
         </Button>
       </PopoverTrigger>
-      <PopoverContent className="w-[300px] p-0">
+      <PopoverContent
+        className="w-[90vw] sm:w-[300px] p-2 sm:p-0"
```

Comment:
See if you can make this work using `--radix-popover-trigger-width` variable (check for examples in codebase), so we don't use custom width here.